### PR TITLE
Bach boolean min max

### PR DESCRIPTION
**Pull request by thijs-obj**
* Support for min() and max() on boolean series
  * min() gives False if there is at least one False in the column
  * max() gives True if there is at least one True in the column
* Enable boolean columns in describe()

This addresses part of issue https://github.com/objectiv/objectiv-analytics/issues/430

### CODE REVIEW

File: bach/bach/describe.py [hunk 1]
```diff
@@ -21,9 +21,8 @@ class SupportedStats(Enum):
     MODE = 'mode'
 
 
-# TODO: Remove this constant when boolean aggregations bug (min, max) is fixed.
```
Comment (thijs-obj) [inline bach/bach/describe.py:24]: I don't quite understand this TODO. We still need this, or am I misunderstanding?

For now I just added SeriesBoolean to the list.

Comment (KathiaBarahona) [reply bach/bach/describe.py:24]: actually we should stop using `_SUPPORTED_SERIES_TYPES`

Comment (thijs-obj) [reply bach/bach/describe.py:24]: discussed out of band. I will do some more work on this

Comment (thijs-obj) [reply bach/bach/describe.py:24]: Follow up PR: https://github.com/objectiv/objectiv-analytics/pull/476

**Review by KathiaBarahona — APPROVED** [review]
LGTM!

**Pull request merged**